File: src/androidTest/java/poetry/android/test/java/data/models/UserTag.java
package poetry.android.test.java.data.models;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

@DatabaseTable
public class UserTag
{
    @DatabaseField(generatedId = true, columnName = "id")
    private int id;

    @DatabaseField(foreign = true, columnName = "user_id")
    private User user;

    @DatabaseField(columnName = "value")
	private String value;

    public String getTag()
    {
        return value;
    }
}

File: src/androidTest/java/poetry/android/test/kotlin/data/models/UserTag.kt
package poetry.android.test.kotlin.data.models

import com.j256.ormlite.field.DatabaseField
import com.j256.ormlite.table.DatabaseTable

@DatabaseTable
class UserTag(
	@DatabaseField(generatedId = true, columnName = "id")
	val id: Int,

	@DatabaseField(foreign = true, columnName = "user_id")
	val user: User,

	@DatabaseField(columnName = "value")
	val tag: String
) {
	internal constructor(): this(0, User.PlaceHolder, "")
}
